lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,28 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  class Car
+    attr_reader :year, :color, :model, :current_speed
+    attr_writer :current_speed
+
+    def initialize(hash)
```

Comment:
`hash` is a bad name for method argument.

Try something like `car_options = {}`